pegasus/sites.v3/hourofcode.com/public/resources/index.moved

```diff
@@ -1 +1 @@
-/resources/promote
+/promote
```

Comment:
Should this go to /promote/resources so it keeps the same content? Or was there a decision to go to /promote since that's the preferred landing page for resources?